services/explorer/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-console */
+import React from 'react';
+import { useMediaQuery } from 'react-responsive';
+import { Button, Header as THeader, NetworkMenu } from '@taraxa_project/taraxa-ui';
+// import { Container } from '@mui/material';
+
+import { HamburgerIcon, TaraxaIcon } from '../icons';
+
+import './header.scss';
+
+const Header = () => {
+  const isMobile = useMediaQuery({ query: `(max-width: 760px)` });
+  const buttons = (
+    <div className="buttonsContainer">
+      <Button label="DAG" color="primary" variant="text" onClick={() => console.log('DAG data')} />
```

Comment:
I would create a Header.effects.ts file, and create there an array of buttonsm each having the props: label, color, onAction etc. Then use that here to iterate over the array. But we can also do this later on.